packages/api/src/routes/medical/dtos/document-bulk-downloadDTO.ts

```diff
@@ -0,0 +1,49 @@
+import { DocumentReference } from "@medplum/fhirtypes";
+import { isMetriportContent } from "../../../external/fhir/shared/extensions/metriport";
+import { capture } from "../../../shared/notifications";
+
+export type DocumentBulkDownloadDTO = {
```

Comment:
this definition needs review, ideally before I go and add docs and modify sdk to minimize back and forth updating. maybe `status` is unnessecary

Reply:
Yeah, I think status is not necessary.

I'd try to make the fields required as much as possible - e.g., AFAIK we always have `mimeType` set, as well as size.